dual-gauge-card.js

```diff
@@ -18,13 +18,20 @@ class DualGaugeCard extends HTMLElement {
     }
     this.config = config;
 
-    if (!this.config.min) {
-      this.config.min = 0;
+    if (!this.config.min & (!this.config.inner.min || !this.config.outer.min)) {
+      this.config.inner.min = this.config.outer.min = 0;
     }
-    if (!this.config.max) {
-      this.config.max = 100;
+    if (!this.config.max & (!this.config.inner.max || !this.config.outer.max)) {
+      this.config.inner.max = this.config.outer.max = 100;
     }
 
+    if (this.config.min) {
```

Comment:
It think the individual value should be more important than the global value.
so please use something like `if (!this.config.inner.min) { this.config.inner.min = this.config.min }` here and revert the changes above.